core/src/View/MDSpan/Kokkos_MDSpan_Extents.hpp

```diff
@@ -63,7 +63,7 @@ struct DimensionFromExtent {
 };
 
 template <>
-struct DimensionFromExtent<std::experimental::dynamic_extent> {
+struct DimensionFromExtent<dynamic_extent> {
```

Comment:
Any particular reason why you decided to qualify at other places and not here?

Reply:
oh woops -- I assume you meant line 57 but that was left in from something earlier so I'll make it consistent

Reply:
This line (66) really is the only one where an unqualified name is used and I also would prefer consistency.

Reply:
Just pushed some changes fixing this

Reply:
You changed it the other way around, though. Now we have two places with unqualified names and all other cases use qualified names.

Reply:
I missed one more, but all the mdspan stuff is now in namespace Kokkos so there is no need to actually qualify. It's only qualified in the tests (and the one more that I accidentally missed).